app/views/admin/attachments/_form.html.erb

```diff
@@ -32,12 +54,38 @@
       <% end %>
     <% end %>
   <% elsif attachment.is_a?(ExternalAttachment) %>
-    <%= form.text_field :external_url %>
+    <div class="govuk-!-margin-bottom-8">
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "External url (required)"
+        },
+        name: "attachment[external_url]",
+        heading_level: 2,
+        heading_size: "l",
+        value: form.object.external_url,
+        error_items: (form.object.errors.as_json(full_messages: true)[:external_url].map do |error|
```

Comment:
same as above regarding helper methods for errors and giving it an id so the error links correctly